### PR TITLE
feature: Add GitLab in pipeline creating

### PR DESCRIPTION
**Pull request by harrisonliu5**
Signed-off-by: harrisonliu5 <harrisonliu_5@163.com>

**What type of PR is this?**

 /kind feature

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes ##1316

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```
/assign @leoendless 
/assign @LinuxSuRen

### CODE REVIEW

File: src/components/Forms/Pipelines/RepoSelect/GitLabForm/index.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of KubeSphere Console.
+ * Copyright (C) 2019 The KubeSphere Console Authors.
```
Comment (shaowenchen) [inline src/components/Forms/Pipelines/RepoSelect/GitLabForm/index.jsx:3]: Copyright (C) 2020

File: src/locales/en/pipeline.js [hunk 2]
```diff
@@ -557,4 +557,6 @@ export default {
   'Brach Run Success': 'Batch Run Completed',
   'Brach Run Fail': 'Batch Run Failed',
   'Run Start': 'Start Running',
+  'Gitlab Server': 'Gitlab Server',
+  Owner: 'Owner',
```
Comment (shaowenchen) [inline src/locales/en/pipeline.js:561]: Why is the Owner not a string key/value, just like 'Owner': 'Owner'?

Comment (harrisonliu5) [reply src/locales/en/pipeline.js:561]: This is the format of js, which can omit the key quotes

File: src/components/Forms/Pipelines/RepoSelect/GitLabForm/index.jsx (hunk 1 above)
Comment (harrisonliu5) [reply src/components/Forms/Pipelines/RepoSelect/GitLabForm/index.jsx:3]: The front end currently defaults to 2019

**Pull request merged**